components/Footer.js

```diff
@@ -0,0 +1,117 @@
+import Image from 'next/image';
+import logo from '../img/logo.png';
+
+export default function Footer() {
+    return(
+      <footer class="bg-neutral-100 text-center text-neutral-600 dark:bg-neutral-600 dark:text-neutral-200 lg:text-left">
+        <div class="scroll-px-0 py-10 text-justify">
+        <div class="grid-1 grid gap-8 md:grid-cols-2 lg:grid-cols-3">
+        <div class="">
+            <Image
+              src={logo}
+              alt="Picture of the author"
+              width={100}
+              height={50}
+            />
+            <p>
+            PyCon Uganda is one of the Python Conferences you can attend in East Africa. We shall talk about Python applications in data, web and other domains. This is a platform where Python users, developers, library developers and companies that use Python come together.
+            This is going to be the FIRST PyCon Conference in Uganda!
+            </p>
+        </div>
+        <div class="">
+            <h6
+            class="mb-4 flex justify-center font-semibold uppercase md:justify-start">
+            Useful links
+            </h6>
+            <p class="mb-4">
+            <a href="#!" class="text-neutral-600 dark:text-neutral-200"
+                >Home</a
+            >
+            </p>
+            <p class="mb-4">
+            <a href="#!" class="text-neutral-600 dark:text-neutral-200"
```

Comment:
@Amanya-Brian how if we added the path for Home as `/` since we are sure `Home Page` will be at `/`

Reply:
```suggestion
            <a href="/" class="text-neutral-600 dark:text-neutral-200"
```

Reply:
Done